src/App.vue

```diff
@@ -307,24 +320,94 @@ async function regenerateAccessToken() {
     // TODO: Regenerate access token functionality
 }
 
-// Warning System functionallity
+// Notification System functionallity
 
 /**
- * Ref which will hold the warnings queue
+ * Ref which will hold the notification queue
  */
-const warnings = ref(useWarningSystem().warningsQueue);
+const notifications = ref(useNotificationSystem().notificationQueue);
 
 /**
- * Computed which will return if there is a warning to show
+ * Computed which will return only a small amount of notifications
  */
-const showWarning = computed(() => !!warnings.value[0].message);
+const displayedNotifications = computed((): Notification[] => {
+    const notificationLimit = 4;
+    return notifications.value.slice(0, notificationLimit);
+});
 
 /**
- * Function which will dismiss the warning
- * @param item The warning to dismiss
+ * Function which will dismiss the notification
+ * @param item The notification to dismiss
  */
-function dismissWarning(item: Warning) {
-    useWarningSystem().removeWarning(item);
+function dismissNotification(item: Notification, index: number) {
+    useNotificationSystem().removeNotification(item);
+    calcMargin(index);
+}
+
+/**
+ * Function which will truncate the text of the notification if it is too long to display
+ * @param index The index of the notification
+ */
+function truncatedText(index: number): string {
```

Comment:
I don't think that this should be our final solution. Using a scrollbar would be better in this case.

Reply:
The current stacking of the notifications doesn't work if the notifications have different sizes thats why i truncated the text

Reply:
I understand that, however the intermediate solution should not be to cut the warning message either.

It seems like putting them below one another does not work at all. Pressing the snackbar button twice results in them overlapping

Reply:
Then you haven't pulled the lates changes i think because for me they aren't stacking anymore

Reply:
ok it doesn't stack but it only shows 4 messages at max at a time. And it is not clear whether there are more messages below. Can you maybe add a scroll-functionality there, so that you're able to scroll through the list of messages? Also maybe cut the last message half-way so that it's clear that there is another message below.

Actually I don't see the necessity to limit the number of messages at all. The user is SUPPOSED to acknowledge the message by closing it, so having an ugly mass of messages on the screen is not an issue imo

Reply:
You want something like this to happen ?

<img width="364" alt="image" src="https://user-images.githubusercontent.com/70775490/234804619-4c1eaa79-eb24-444a-b211-20ff6b9626b3.png">

Reply:
yes, I think this would actually be the better solution

Reply:
OK I just commited the change

Reply:
just without the truncation tho

Reply:
About the text truncing. I have already made a new issue #130 to make it possible to have different sizes of the notification and maybe icons etc.

And because we do not have any system using the notifications we still have to implement it into the other systems, and then we will maybe also run into things that we want to displayer or if we really need that much displayed text, like its a notification for the user and not a wikipedia page of some sorts

Reply:
Without truncation and that the notification is currently set to a specific height it would look like this:

<img width="296" alt="image" src="https://user-images.githubusercontent.com/70775490/234811009-4b1a5b2e-9200-4c15-81f5-4857b62373bd.png">

And tbh we didn't even decided even make a mock-up for how it should or could look maybe we should do something like this bevor we discuss on how we want it to look.